5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { mockTodo } from "../../../test/utils";
+import { TodoListItemDriver } from "./TodoListItem.driver";
+
+describe("todo list item component", () => {
+  let driver : TodoListItemDriver;
+
+  beforeEach(() => {
+    driver = new TodoListItemDriver();
+  });
+
+  test("should display description text", async () => {
+    const item = { text: "test" };
+    driver.given.item(item);
+
+    driver.when.render();
+
+    expect(driver.get.todoText()).toEqual(item.text);
+  });
+
+
+  test("should display switchMode status", async () => {
```

Comment:
test description isn't very clear

Reply:
changed to "should verify that a new item status is unfinished by default".
Actually, I'm not sure if this test is valuable, that do you think?